client/jetpack-cloud/sections/partner-portal/billing-details/index.tsx

```diff
@@ -11,15 +11,16 @@ export default function BillingDetails(): ReactElement {
 	const translate = useTranslate();
 	const moment = useLocalizedMoment();
 	const billing = useBillingDashboardQuery();
+	const useDailyPrices = billing.isSuccess && billing.data.priceInterval === 'day';
```

Comment:
@atanas-dev small note after the refactor that doesn't make any real difference - but if you (or I) touch the billing details list in the future, then we can remove the two instances that uses `billing.isSuccess && useDailyPrices ...` as a condition since `useDailyPrices` already contains `billing.isSuccess` 😄